### PR TITLE
eve/http: add request/response headers test

### PR DESCRIPTION
**Pull request by maurizioabba**
- added readme
- removed base include suricata yaml
- added min-version: 5.0.0

### CODE REVIEW

File: tests/http-all-headers/check.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+. ${TOPDIR}/util/functions.sh
+
+# 9 request headers (2 rows per header + start and end of list).
+n=$(jq '.http.request_headers | length' eve.json)
+assert_eq 9 "$n"
+
+# Simplified test: check 1 request header
+# Arbitrary decision to check "Host": "www.ethereal.com"
+n=$(jq ".http.request_headers[] | select(.name | contains(\"Host\")) | select(.value | contains(\"www.ethereal.com\"))" eve.json)
```
Comment (victorjulien) [inline tests/http-all-headers/check.sh]: I think we should be able to have this expressed in the test.yaml itself, w/o the need for a jq wrapper script.

**Comment by victorjulien** [comment]
@shivan1b could you pick up this PR? Maurizio won't be able to anymore.

**Comment by shivan1b** [comment]
@victorjulien yes sure. Do we have a redmine ticket for this?

**Comment by victorjulien** [comment]
Not for the test, but the functionality was tracked in https://redmine.openinfosecfoundation.org/issues/2563

**Comment by jasonish** [comment]
Closing. Continued at https://github.com/OISF/suricata-verify/pull/149 with jq tests removed.

**Pull request closed without merging**